frontend/src/utils/getUrl.ts

```diff
@@ -1,11 +1,11 @@
-export const getUrl = (path: string, params?: Record<string, string | undefined>) => {
+export const getUrl = (path: string, params?: Record<string, string | number | undefined>) => {
     const protocol = window.location.protocol
     const host = window.location.host
     const url = new URL(`${protocol}//${host}${path}`)
     if (params)
         Object.keys(params).forEach((param) => {
             const value = params[param]
```

Comment:
Do you know about `Object.entries`?

Reply:
Yes, sure. Sometimes I just prefer this one. Looks more connected visually.